internal/server/models/passwordresettoken.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+import (
+	"time"
+
+	"github.com/infrahq/infra/uid"
+)
+
+type PasswordResetToken struct {
```

Comment:
To be clear, this _isn't_ a database model correct?

Reply:
it is. why?